packages/datadog-plugin-cucumber/src/index.js

```diff
@@ -30,26 +33,35 @@ class CucumberPlugin extends Plugin {
 
     const testEnvironmentMetadata = getTestEnvironmentMetadata('cucumber', this.config)
     const sourceRoot = process.cwd()
+    const codeOwnersEntries = getCodeOwnersFileEntries(sourceRoot)
 
     this.addSub('ci:cucumber:run:start', ({ pickleName, pickleUri }) => {
       const store = storage.getStore()
       const childOf = store ? store.span : store
       const testSuite = getTestSuitePath(pickleUri, sourceRoot)
 
+      const testSpanMetadata = {
+        [SPAN_TYPE]: 'test',
+        [RESOURCE_NAME]: pickleName,
+        [TEST_TYPE]: 'test',
+        [TEST_NAME]: pickleName,
+        [TEST_SUITE]: testSuite,
+        [SAMPLING_RULE_DECISION]: 1,
+        [SAMPLING_PRIORITY]: AUTO_KEEP,
+        [TEST_FRAMEWORK_VERSION]: this.tracer._version,
+        ...testEnvironmentMetadata
+      }
+
+      const codeOwners = getCodeOwnersForFilename(testSuite, codeOwnersEntries)
```

Comment:
sadly we can't add tests for code owners here because of the way we run `cucumber`. Since we set `cwd` to the folder in `/versions`, the `testSuite`s end up being relative paths, something like `../../../datadog-plugin-cucumber/test/features/simple.feature`, which can't be used to compare against in the `CODEOWNERS`. 

This shouldn't be an issue for customers that are running `cucumber` normally though.